conf/gce.yaml.template

```diff
@@ -1,12 +1,8 @@
 GCE:
   # Google Provider as Compute Resource
-  # Project name in Google provider
-  PROJECT_ID:  # example-project-id
-  # Service Account email id which has compute admin permission
-  CLIENT_EMAIL: client@example.com
   # client json Certificate path which is local path on satellite
   CERT_PATH: /path/to/certificate.json
   # Zones
   ZONE: example-zone
-  # client certificate URL
-  CERT_URL: /path/to/certificate/url
+  # client certificate
+  CERT: "{}" # client json Certificate
```

Comment:
Looks like you were able to sort out the default here for the value that json will parse correctly from the yaml?